docs/modules/ROOT/pages/how-to-guides/testing_applications/enable_snyk_check_for_a_product.adoc

```diff
@@ -0,0 +1,81 @@
+
+== Enabling Snyk task
+
+. Register for a Snyk account or log in at https://app.snyk.io/.
+. Get a Snyk token.
+.. In the lower left of the home page, click your name, then select *Account settings*.
+.. From the Account Settings page, select *General*, which is the default, then *Auth Token*.
+.. Under the *Auth Token* section, click on *Click to View* button to see the *KEY* value of the automatically generated token.
+. To enable Snyk code, go to *Settings* on the left panel > *Snyk Code* > *Enable Snyk Code* > click the *Disabled* bar.
+.. Save the changes when it is changed to *Enabled*.
+. Add your new secret *KEY* value to your workspace.
+    .. Login to the console https://console.redhat.com/preview/application-pipeline.
+    .. From the left menu, click *Secrets*.
+    .. Click *Add secret*.
+    .. The *Add secret* page displays options for your new secret. Specify the following:
+    ... Select Secret for *Build*.
+    ... From the *Secret type* drop-down menu, choose *Key/value secret*.
+    ... From the *Secret name* drop-down menu, select *snyk-secret*.
+    ... Under *Key/value secret*, expand *Key/value 1* section the *Key* name is automatically populated as *snyk-token*.
+    ... Paste your Snyk token into the *Upload the file with value for your key or paste its contents* field.
+    ... Click *Add secret* to save it.
+. Update your *.tekton* YAML files to specify your Snyk secret.
+.. From your GitHub repository, update the *pull-request.yaml* files in your *.tekton* directory; for example: link:https://github.com/sonam1412/devfile-sample-python-basic/blob/main/.tekton/devfile-sample-python-basic-iea1-pull-request.yaml[devfile-sample-python-basic/.tekton/devfile-sample-python-basic-rhq8-pull-request.yaml]
+.. Add new lines to the code in your YAML files to include *snyk-secret*, as shown in the following example:
++ 
+.Example:
++
+[source,yaml]
+--
+    - default: "false"
+      description: Execute the build with network isolation
+      name: hermetic
+      type: string
+    - default: ""
+      description: Build dependencies to be prefetched by Cachi2
+      name: prefetch-input
+      type: string
+    - default: "false"
+      description: Java build
+      name: java
+      type: string
+    - default: "snyk-secret"
+      description: Snyk Token Secret Name
+      name: snyk-secret
+      type: string
+--
+
+You've enabled the Snyk task for your build pipeline.
+
+== Obtaining a Red Hat Snyk token using a service account
+
+[discrete]
+=== Create an organization-level Snyk token
+
+. Register for a Snyk account or log in at https://app.snyk.io/.
+. Under *Group* in the left navigation panel, select *Red Hat*.
+. Under *Organization* in the same panel, select the name of your organization.
+. Select *Members* to identify who from your organization has *Org Admin* access. Only those members with *Org Admin* access can create a service account and token.
```

Comment:
now that i'm looking at this again, could you clarify why the user should select Members? is it to check their own permissions or to find a teammate who has the right permissions?

depending on your answer, we might need to switch the order of steps 4 and 5.

Reply:
It is to check whoever in the group has Org Admin access to create the secret. I have reworded that a little bit. Please check if that make sense.